R/zchunk_LB1322.Fert.R

```diff
@@ -1,31 +1,33 @@
 #' module_energy_LB1322.Fert
 #'
-#' Briefly describe what this chunk does.
+#' This chunk computes fertilizer production and energy inputs by technology.
 #'
 #' @param command API command to execute
 #' @param ... other optional parameters, depending on command
 #' @return Depends on \code{command}: either a vector of required inputs,
 #' a vector of output names, or (if \code{command} is "MAKE") all
 #' the generated outputs: \code{L1322.Fert_Prod_MtN_R_F_Y}, \code{L1322.IO_R_Fert_F_Yh}, \code{L1322.in_EJ_R_indenergy_F_Yh}, \code{L1322.in_EJ_R_indfeed_F_Yh}, \code{L1322.Fert_NEcost_75USDkgN_F}. The corresponding file in the
 #' original data system was \code{LB1322.Fert.R} (energy level1).
-#' @details Describe in detail what this chunk does.
+#' @details Fertilizer production and energy information was calculated using country-level fuel share and energy intensity data
+#' to allocate generic fertilizer production data across technologies. Common sense approaches were taken to ensure no unrealistic information was
+#' reported at the country level before aggratating to the regional level. Also, a final cost table was calculated using USA market fertilizer
+#' price data and H2A characteristics of hydrogen production technologies.
 #' @importFrom assertthat assert_that
```

Comment:
I coincidentally know that the H2A is the DOE EERE Hydrogen program's analysis model.  The inputs in the level 0 file are based on a scenario using a version of that model for certain parameters potentially a long time ago.  You can add a citation, even a web link to that program (https://www.hydrogen.energy.gov/h2a_analysis.html)  but you would have to ask Page about any more specifics.  Like, do we have the model saved with those inputs, or is it from a long time ago.  What to do..., we can define H2A and provide a link.  Ask Page if we know anything more about it.